claim/README.md

```diff
@@ -80,29 +80,29 @@ you don't see the node in your Space after 60 seconds, see the [troubleshooting
 
 ### Claim an Agent running in Docker
 
-The claiming process works with Agents running inside of Docker containers. You can use `docker exec` to run the
-claiming script on containers already running, or append the claiming script to `docker run` to create a new container
-and immediately claim it.
+The claiming process works with Agents running inside of Docker containers. You can either set a specific set
+of environment variables in the container to have it automatically claimed on startup or restart, or you can use
+`docker exec` to manually claim an already-running container.
```

Comment:
```suggestion
The claiming process works with the Netdat Agent running inside of a Docker container. Use a specific set
of environment variables in the container to have it automatically claimed on startup or restart, or use
`docker exec` to manually claim an already-running container.
```

Reply:
I’m curious about the reasoning behind shifting to a more imperative wording here.

Reply:
Lately I've been more critical about Netdata's writing—maybe it's the time on spent on the style guide—and feel we should be more active in our tone. I prefer "do X or Y" over "you can do X or Y" because it's shorter and feels more empowering and inclusive.

We're prolific in our use of "you can," from the beginning of the project. I don't know if I inherited it from the existing docs or brought more of the tendency in myself, but I'm trying to correct myself and move docs away from that style as I approach it. I even wrote a Go script to parse the docs and list every one that uses "you can," among other things...

Reply:
Also, I typo'd **Netdata** in my suggestion. Whoops.

Reply:
OK, the shorter part I get, but I personally feel that using `do X or Y` sounds out of place in documentation unless you use it in a way that doesn’t result in it sounding like a command.

IOW, this should probably be ‘To claim an instance of the Netdata Agent running in a Docker container, either do X or do Y’, otherwise it sounds like we’re commanding the reader to do something instead of explaining how to do it. The ‘either/or’ aspect is also kind of important here given how long each of the optional clauses is in the sentence, as it solidifies ahead of time that this is a set of choices.